packages/sonataflow-image-common/resources/modules/kogito-maven/common/module.yaml

```
@@ -23,16 +23,20 @@ version: "main"
 envs:
   - name: "MAVEN_VERSION"
     description: "The Maven version to setup with this module"
+    value: "3.8.6"
   - name: "MAVEN_HOME"
     value: "/usr/share/maven"
   - name: "MAVEN_CMD"
-    value: "/usr/share/maven/bin/mvnw"
+    value: "/usr/share/maven/bin/mvn"
   - name: "MAVEN_SETTINGS_PATH"
     description: "The location of the settings.xml file"
     value: "${KOGITO_HOME}/.m2/settings.xml"
   - name: "MAVEN_CONTAINER_BUILD_SETTINGS_PATH"
     description: "The location of the settings.xml file during container image build"
     value: "${KOGITO_HOME}/.m2/maven-m2-repo-via-http-settings.xml"
+  - name: "KOGITO_MAVEN_HOME"
```

```suggestion
  - name: "MAVEN_WRAPPER_HOME"
```
I think this name improves understandability. 😅

---

So change it to `ALTERNATIVE_MAVEN_HOME`. I was confused by the `mvnw`. I guess it was from an old commit.